snapshot-labs/sx-monorepo
# feat: add support for single choice and approval offchain proposal  creation

### Summary



Closes: #70 

### How to test

1. Go to a offchain where you have proposal creation voting power
2. Go to proposal creation page
3. Select an other proposal type other than the default `basic`
4. It should show a choices customization form
5. Set and add as many choices as you want
6. Submit the form
7. It should create a proposal with the chosen vote type and choices

### To-Do

- [x] Add vote type selector to proposal editor
- [x] Add choices customisation to proposal editor
